File: witch-apps/app-api/src/main/java/com/ssafy/witch/controller/group/response/GroupResponse.java
package com.ssafy.witch.controller.group.response;

import com.fasterxml.jackson.annotation.JsonInclude;
import com.ssafy.witch.controller.user.response.UserResponse;
import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
@Getter
public class GroupResponse {

  private String groupId;
  private String name;
  private String groupImageUrl;

  @JsonInclude(JsonInclude.Include.NON_NULL)
  private String createdAt;

  @JsonInclude(JsonInclude.Include.NON_NULL)
  private UserResponse leader;

}

File: Witch/app/src/main/java/com/ssafy/witch/data/model/response/GroupResponse.kt
package com.ssafy.witch.data.model.response

data class GroupResponse(
    val cntLateArrival: Int,
    val groupId: String,
    val groupImageUrl: String,
    val isLeader: Boolean,
    val name: String
){
    constructor(groupId: String, groupImageUrl: String,name: String): this(0, groupId, groupImageUrl, false, name)
}